devicon.json

```diff
@@ -2835,6 +3140,9 @@
     },
     {
         "name": "jenkins",
+        "altnames": [
+            "hudson"
```

Comment:
This might be confusing/irretating or?

Reply:
Jenkins is a fork of hudson. So hudson is the old name, but they had to change it due to legal issues with Oracle.

You can read more about it here: https://www.javatpoint.com/jenkins-vs-hudson#:~:text=Jenkins%20vs%20Hudson%20Comparison%20Table,be%20forked%20and%20Jenkins%20CI.

Reply:
So imo it should stay, but guess it's up for discussion